shell/platform/fuchsia/flutter/platform_view_unittest.cc

```diff
@@ -143,13 +147,25 @@ class MockPlatformViewDelegate : public flutter::PlatformView::Delegate {
   const flutter::ViewportMetrics& metrics() const { return metrics_; }
   int32_t semantics_features() const { return semantics_features_; }
   bool semantics_enabled() const { return semantics_enabled_; }
+  const std::vector<std::unique_ptr<flutter::PointerDataPacket>>&
+  pointer_packets() const {
+    return pointer_packets_;
+  }
+  // Unlike the other getters, this one hands data ownership to the caller.
+  std::vector<std::unique_ptr<flutter::PointerDataPacket>>
+  obtain_pointer_packets() {
+    auto tmp = std::move(pointer_packets_);
+    pointer_packets_.clear();
+    return tmp;
+  }
 
  private:
   std::unique_ptr<flutter::Surface> surface_;
   std::unique_ptr<flutter::PlatformMessage> message_;
   flutter::ViewportMetrics metrics_;
   int32_t semantics_features_ = 0;
   bool semantics_enabled_ = false;
+  std::vector<std::unique_ptr<flutter::PointerDataPacket>> pointer_packets_;
```

Comment:
Nit: Please move above w/ message_ & metrics_.  Keep primitive types at the end (this gives both readability and alignment advantages).  This is a rule I try to follow, not a google style thing